metrics/web/lib/common/domain/entities/instant_config.dart

```diff
@@ -1,8 +1,7 @@
-import 'package:meta/meta.dart';
+import 'package:equatable/equatable.dart';
 
 /// An entity representing an instant config.
-@immutable
-class InstantConfig {
+class InstantConfig extends Equatable {
```

Comment:
As this is equatable now, should we rethink some test cases related to the domain layer?